sdk/script/management/account-manager.ts

```diff
@@ -410,14 +313,13 @@ export class AccountManager {
     public addApp(appName: string): Promise<App> {
         return Promise<App>((resolve, reject, notify) => {
             var app = <App>{ name: appName };
-            var requester = (this._authedAgent ? this._authedAgent : request);
 
-            var req = requester.post(this.serverUrl + "/apps/");
-            this.attachCredentials(req, requester);
+            var request: superagent.Request<any> = superagent.post(this._serverUrl + "/apps/");
+            this.attachCredentials(request);
 
-            req.set("Content-Type", "application/json;charset=UTF-8")
+            request.set("Content-Type", "application/json;charset=UTF-8")
```

Comment:
Are we using this content type enough to justify making it a constant somewhere?